### DISCUSSION TITLE
Latest Play store release

### DISCUSSION THREAD

**Discussion by markcs** [bitfireAT/icsx5#245] [General]
Hi,

I was checking Google Play, and the latest update available is from March 2023.

I notice there have been a few releases from the project page.  I can not install apks, so does anyone know when these releases will be pushed also to Google Play?

Thanks

**Comment by sunkup** [comment]
We are currently waiting for google to verify the latest version (v2.2). Only google knows when it will be available.

**Reply by markcs** [reply]
Okay. Thanks. Good to know it is on its way!
